package.json

```diff
@@ -9,7 +9,29 @@
     "@aragon/apps-voting": "2.0.0"
   },
   "devDependencies": {
-    "@aragon/cli": "^5.4.0"
+    "@aragon/cli": "^5.4.0",
+    "@aragon/test-helpers": "^1.0.1",
+    "@babel/core": "^7.2.2",
+    "@babel/plugin-proposal-class-properties": "^7.2.3",
+    "@babel/polyfill": "^7.2.5",
+    "@babel/preset-env": "^7.2.3",
+    "babel-eslint": "^10.0.1",
+    "babel-plugin-styled-components": "^1.10.0",
+    "eslint": "^5.13.0",
+    "eslint-config-prettier": "^4.0.0",
+    "eslint-config-standard": "^12.0.0",
+    "eslint-config-standard-react": "^7.0.2",
+    "eslint-plugin-import": "^2.16.0",
+    "eslint-plugin-jest": "^22.2.2",
+    "eslint-plugin-node": "^8.0.1",
+    "eslint-plugin-prettier": "^3.0.1",
+    "eslint-plugin-promise": "^4.0.1",
+    "eslint-plugin-react": "^7.12.4",
+    "eslint-plugin-standard": "^4.0.0",
+    "parcel-bundler": "^1.11.0",
+    "prettier": "^1.16.4",
+    "solidity-coverage": "^0.5.11",
+    "solium": "^1.2.3"
```

Comment:
Probably best to move this to `eth-lint`

Reply:
Done.